tools/publish.sh

```diff
@@ -30,6 +30,7 @@ crates=(
     bevy_wgpu
     bevy_internal
     bevy_dylib
+    bevy_geometry
```

Comment:
According to the Comment at the top of the file, it needs to be placed at least before `bevy_internal`.
I would suggest placing it right after `bevy_math`, incase another crate depends on` bevy_geometry`.